Doc/howto/functional.rst

```diff
@@ -529,18 +517,15 @@ Passing values into a generator
 
 In Python 2.4 and earlier, generators only produced output.  Once a generator's
 code was invoked to create an iterator, there was no way to pass any new
-information into the function when its execution is resumed.  You could hack
-together this ability by making the generator look at a global variable or by
-passing in some mutable object that callers then modify, but these approaches
-are messy.
+information into the function when its execution is resumed.
```

Comment:
It's important to be precise at all times in documentation. By deleting most of this paragraph, it seems to me you make the article less precise. You now state that "there was no way to pass any new information into the function when its execution is resumed", but as the existing text points out, this is false -- there _were_ ways of doing this, they just weren't particularly elegant.

Reply:
Good point, I'll reword it.

Reply:
Thanks, I like your rewording!